rules/macos/process_creation/proc_creation_macos_payload_decoded_and_decrypted.yml

```diff
@@ -0,0 +1,31 @@
+title: Payload Decoded and Decrypted via Built-in Utilities
+id: 234dc5df-40b5-49d1-bf53-0d44ce778eca
+status: experimental
+description: Detects when a built-in utility is used to decode and decrypt a payload after a macOS disk image (DMG) is executed. Malware authors may attempt to evade detection and trick users into executing malicious code by encoding and encrypting their payload and placing it in a disk image file. This behavior is consistent with adware or malware families such as Bundlore and Shlayer.
+author: Tim Rauch (rule), Elastic (idea)
+references:
+    - https://github.com/elastic/protections-artifacts/commit/746086721fd385d9f5c6647cada1788db4aea95f#diff-5d42c3d772e04f1e8d0eb60f5233bc79def1ea73105a2d8822f44164f77ef823
+date: 2022/10/17
+logsource:
+    category: process_creation
```

Comment:
Don't have mac but as it use Darwin as core don't not a sysmon to have process_creation ?

Reply:
Similar to the Linux `process_creation` rule. These are here for people who have a way to collect these types of events and maybe for us when we have aurora for MAC also.

Reply:
Yes , just a open question and how the rule was tested.